pytorch_forecasting/utils/_utils.py

```diff
@@ -429,7 +429,16 @@ def move_to_device(
         x on targeted device
     """
     if isinstance(device, str):
-        device = torch.device(device)
+        if device == "mps":
+            if hasattr(torch.backends, device):
+                if torch.backends.mps.is_available() and torch.backends.mps.is_built():
+                    device = torch.device(device)
+                else:
+                    device = torch.device("cpu")
+            else:
```

Comment:
is this perhaps too aggressive? Are we moving devices to cpu that couId work?

Reply:
Yeah, I agree we can simplify and remove the setting to CPU as it is always the default device.

Reply:
I mean, should we change the device not *only* if (a) it is mps, and (b), mps is not available?

Reply:
I think the current branching makes sense - not sure if you changed it or if I now just understod it, but I think it is exhaustive and the step-out covers only the MPS-not-present case